lib/ruby_llm/model_info.rb

```diff
@@ -15,7 +15,7 @@ module RubyLLM
   class ModelInfo
     attr_reader :id, :created_at, :display_name, :provider, :metadata,
                 :context_window, :max_tokens, :supports_vision, :supports_functions,
-                :supports_json_mode, :input_price_per_million, :output_price_per_million, :type, :family
```

Comment:
Is `json_mode` not a separate thing from structured output (albeit related)?

Reply:
Yeah it is kind of, but I don't see us using it anywhere in the code. So I decided to remove/replace it for now. Thoughts?

Reply:
True. Good point. Might want to add the functionality later.

Reply:
I agree with removing that.